websocket/config.go

```diff
@@ -4,13 +4,29 @@ import (
 	"errors"
 )
 
+// Config is a set of WebSocket client options.
 type Config struct {
-	APIKey     string
-	Feed       Feed
-	Market     Market
+	// APIKey is the API key used to authenticate against the server.
+	APIKey string
```

Comment:
Should this be private for security purposes? Is there any need for external use of it?

Reply:
it needs to be public so users can pass it to us. i don't think there are any security implications to setting a field here

Reply:
Never-mind, this is the config struct. Disregard.